### PR TITLE
[Do Not Merge] Docker: Added docker containers for easy deployment

### PR DESCRIPTION
**Pull request by deshraj**

### CODE REVIEW

File: docker/nodejs/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
+FROM nginx:latest
+
+########### Nginx Configuration starts here ###########
```
Comment (dexter1691) [inline docker/nodejs/Dockerfile]: why is there an nginx configuration in nodejs? 

Shouldn't this be a separate container with nginx linking both node.js and django specific containers.

Comment (deshraj) [inline docker/nodejs/Dockerfile]: Yeah, that's the plan. The PR is not fully ready. I pushed so that I don't loose the code and get comments from others.

Comment (dexter1691) [inline docker/nodejs/Dockerfile]: SG.

**Comment by deshraj** [comment]
Closing this as this is duplicate as #323

**Pull request closed without merging**